package.json

```diff
@@ -104,6 +104,7 @@
     "decode-uri-component@^0.2.0": "^0.2.1",
     "node-fetch@2.6.0": "^2.6.7",
     "node-fetch@2.6.1": "^2.6.7",
-    "trim@0.0.1": "^0.0.3"
+    "trim@0.0.1": "^0.0.3",
+    "@graphql-tools/links@npm:8.3.36/form-data": "^4.0.4"
```

Comment:
Latest version of @graphql-tools/links [9.0.29](https://www.npmjs.com/package/@graphql-tools/links/v/9.0.29) has `form-data` v4.0.0